include/faabric/util/json.h

```diff
@@ -1,23 +1,19 @@
 #pragma once
 
-#include <faabric/proto/faabric.pb.h>
 #include <faabric/util/exception.h>
 
+#include <google/protobuf/util/json_util.h>
+
 namespace faabric::util {
-std::string messageToJson(const faabric::Message& msg);
+std::string messageToJson(const google::protobuf::Message& msg);
 
-faabric::Message jsonToMessage(const std::string& jsonIn);
+void jsonToMessage(const std::string& jsonStr, google::protobuf::Message* msg);
```

Comment:
I need to change this signature, as we can not return an instance of the abstract class `google::protobuf::Message`.

Instead, we take the argument as a pointer.